docs/tools/vdb_table/data/neo4j.json

```diff
@@ -14,26 +14,27 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "Java",
+      "Scala"
     ],
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 0,
-  "vector_launch_year": 0,
+  "github_stars": 12200,
```

Comment:
The github stars are for open-source repo for the vector search engine/db itself. Please reset this. Thanks.

Reply:
Shouldn't there also be a `source_url` here to show what repo we're referring to?